### PR TITLE
Porfolio: Popup window

### PR DESCRIPTION
**Pull request by mateoburvi**
# Additions

In this part, we added

- JS for Project creation
- CSS for Popup Windows

Note: I had to make a new branch since my last one wasn't working properly, in this one I wont have commits from my partner, but I had some in previous branches

This was the last pull request, it shows the commits made by my coding partner
https://github.com/mateoburvi/Portfolio/pull/10

Coding partner commits

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @mateoburvi ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly ensure your coding partner contributes to this milestone since it is a pair programming activity. 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js
Comment (Efiamotu-1) [inline main.js:1]: - [ ] The posititon of the close button on figma and on your project do not match please kindly ensure that your project design matches completely the provided figma design for the popup :+1: 
- [ ] As shown by the circled button for `see live` & `see source` this is how they are to be displayed on desktop view. However your current design doesnt match this. please kindly make the necessary corrections :+1: :sparkles: 
- figma design
dnm
- your designs
icb
iicb

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @mateoburvi , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -369,6 +369,185 @@ button:active {
...
+  grid-area: 1/1/2/2;
+  align-self: start;
+  justify-self: end;
+  margin: 1rem;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+  position: relative;
+}
+
```
Comment (KanzaTahreem) [inline style.css:424]: - [ ] Good job so far 👍👍 However, as suggested by the previous reviewer [here](https://github.com/mateoburvi/Portfolio/pull/12#discussion_r1249221892), kindly make sure to place the close button in accordance with the given Figma template✨⛏ to get a design close to the Figma template as much as possible ✌


Your design | Figma design 
---| ---|
image | image

File: style.css [hunk 2]
```diff
@@ -678,46 +841,74 @@ label {
...
+  }
+
+  .popup__buttons__container {
+    width: 100%;
+    grid-area: tags;
+    flex-flow: row wrap;
+    justify-content: flex-end;
+    align-self: center;
+    margin-top: 2.5rem;
+  }
```
Comment (KanzaTahreem) [inline style.css]: - [ ] Similarly, for the project button, kindly make sure to place them according to the given design guidelines as suggested [here](https://github.com/mateoburvi/Portfolio/pull/12#discussion_r1249221892) to get a design close to the Figma template as much as possible 💪📌

Your design | Figma design 
---| ---|
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @mateoburvi 

Good job so far! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (mateoburvi) [reply style.css:424]: can i have your slack? i dont have much time. please just help me approve this block and I will correct the mistakes later

File: style.css [hunk 3]
```diff
@@ -369,6 +369,185 @@ button:active {
...
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: var(--backgroundColor);
+  background-blend-mode: multiply;
+  z-index: 100;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
```
Comment (1GORDON) [inline style.css:386]: Great job @mateoburvi 🤝

- [ ] As requested by the previous reviewer [here](https://github.com/mateoburvi/Portfolio/pull/12#discussion_r1249247967), kindly make sure to place the close button in accordance with the given Figma template✨⛏ to get a design close to the Figma template as much as possible ✌

File: style.css [hunk 4]
```diff
@@ -369,6 +369,185 @@ button:active {
...
+  width: 90%;
+  max-width: 920px;
+  max-height: 1083px;
+  margin: 5% auto 2% auto;
+  background: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 8px;
+  z-index: 200;
+}
```
Comment (1GORDON) [inline style.css:399]: Great job @mateoburvi 🤝

- [ ] As requested by the previous reviewer [here](https://github.com/mateoburvi/Portfolio/pull/12#discussion_r1249248370), for the project button, kindly make sure to place them according to the given design guidelines as suggested  to get a design close to the Figma template as much as possible 💪📌

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @mateoburvi ✋

Good job so far!

## Highlights!
- GitHub Flow well used ✔️ 
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**